snap/snapcraft.yaml

```diff
@@ -68,14 +76,22 @@ parts:
 
   dnsmasq:
     plugin: make
-    source: https://git.launchpad.net/ubuntu/+source/dnsmasq
-    source-type: git
-    source-branch: applied/2.79-1ubuntu0.7
+    source: .  # IGNORE: will be replace by ESM dnsmasq source package, but local-dir is faster than git checkout
```

Comment:
In other cases we pull from places like https://git.launchpad.net/ubuntu/+source/<source_pkg> , not sure if we have something similar for ESM. I'm fine with getting the sources with apt-get if trying to get from a git repo complicates things.

Reply:
Yes, those are the git branches from the `git-ubuntu` importer. I'm not aware that it would also handle ESM packages. Those seem to be available in the restricted private-ppa only.

Reply:
FTR: Security Engineering (@dodys) says there's no such git banches for the ESM packages.